lua/luau-lsp/studio.lua

```diff
@@ -0,0 +1,141 @@
+local compat = require "luau-lsp.compat"
+local config = require "luau-lsp.config"
+local http = require "luau-lsp.http"
+local uv = compat.uv
+
+-- Acknowledgements:
```

Comment:
This should be at the top of the file.

Reply:
Do you mean the Acknowledgements or compat.uv?

Reply:
The acknowledgements.